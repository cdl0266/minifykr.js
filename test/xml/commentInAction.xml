<krpano>
	<!-- Outter Comment 1 -->
	<include url="commentInAction2.xml"/>

	<action name="commentinaction">
	set(x, y);
	<!-- Comment in action -->
	set(z, w);
	</action>
</krpano>