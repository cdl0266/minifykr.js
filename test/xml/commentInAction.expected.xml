<krpano>
	<!-- Outter Comment 1 -->
	<!-- Outter Comment 2 -->
	<action name="commentinaction2">
	<!-- Comment in action2.0 -->
	set(u, i);
	<!-- Comment in action2.1 -->
	set(h, j);
	<!-- Comment in action2.3 -->
	</action>

	<action name="commentinaction">
	set(x, y);
	<!-- Comment in action -->
	set(z, w);
	</action>
</krpano>